deploy/layered-sre-authorization/rhmi/02-layered-cs-sre-admin.SubjectPermission.yaml

```diff
@@ -10,7 +10,7 @@ spec:
   permissions:
   - allowFirst: true
     clusterRoleName: layered-cs-sre-admin-project
-    namespacesAllowedRegex: .*
+    namespacesAllowedRegex: (^redhat-rhmi$|^redhat-rhmi-.*)
     namespacesDeniedRegex: (^kube$|^kube-.*|^openshift$|^openshift-.*|^default$)
```

Comment:
Based on my understanding, we don't need an allow and a deny.  see: https://github.com/openshift/managed-cluster-config/pull/545/files.  We will also need rhoam namespaces: "(^redhat-rhmi-.*|^redhat-rhoam-.*)"

Reply:
The namespace prefix Regix should be `(^redhat-rhmi$|^redhat-rhmi-.*)` and `(^redhat-rhoam$|^redhat-rhoam-.*)`. We should explicitly deny all first and then allow, this is best practice here.

Reply:
We can't do this in RHMI. It must remain as a `.*`. RHMI has components that dynamically creates namespaces based on what the customer creates. (In fuse AFAIK).